app/models/organisation.rb

```diff
@@ -18,8 +18,62 @@ def title
     @content_item.content_item_data["title"]
   end
 
+  def logo
+    details["logo"]
+  end
+
+  def base_path
+    @content_item.content_item_data["base_path"]
+  end
+
+  def ordered_featured_links
+    details["ordered_featured_links"]
+  end
+
+  def body
+    details["body"]
+  end
+
+  def ordered_featured_documents
+    details["ordered_featured_documents"]
+  end
+
+  def child_organisation_count
+    links["ordered_child_organisations"].count
+  end
+
+  def ordered_featured_policies
+    links["ordered_featured_policies"]
+  end
+
+  def social_media_links
+    details["social_media_links"]
+  end
+
+  def organisation_type
+    details["organisation_type"].tr("_", " ")
+  end
+
+  def ordered_ministers
+    details["ordered_ministers"]
+  end
+
+  def board_members
+    details["ordered_board_members"]
+  end
+
+  def ordered_contacts
+    links["ordered_contacts"]
+  end
+
+  def ordered_corporate_information_pages
+    details["ordered_corporate_information_pages"]
+  end
+
   def ordered_parent_organisations
-    links["ordered_parent_organisations"]
+    if links["ordered_parent_organisations"].present?
```

Comment:
you can leave the if statement as the result of this method will be nil when the key is not present.
Which is the same as doing `links["ordered_parent_organisations"]`

example:

```ruby
irb(main):001:0> p = {}
=> {}
irb(main):002:0> p["a"]
=> nil
```

Reply:
i have removed the statement out of here and put it in the view instead as some orgs don't have a parent organisation